### ISSUE TITLE
Decision on display removal of Makerspace from central ULS-Library

### ISSUE THREAD

**Issue by manolobevia** [cul-it/uls#1034]
@sew268 @eeh53 Right now, we are displaying Makerspace as unit on `https://live-uls-library-cornell-edu.pantheonsite.io/staff/`. I think it should be removed  since Makerspace is not a unit library. What do you think?

**Comment by cappadona** [comment]
@manolobevia Good question, but not a high priority issue since we're not going live with the central site (all staff) anytime soon, correct?

Just want to clarify that this is not causing any issues on the [Mann site](https://mann.library.cornell.edu/staff-profiles), where I still see all 4 cross-listed Makerspace staff 😄

**Comment by cappadona** [comment]
Looking at this again, I think the bigger problem is that we have duplicate staff in the central list (and it's not limited to just the Makerspace). Maybe that's what you're referring to?

**Comment by manolobevia** [comment]
No, I am referring to Makerspace display in the facet side bar. ![Screen Shot 2021-05-14 at 10 55 33 AM](https://user-images.githubusercontent.com/8395482/118298228-349d5480-b4ad-11eb-89c5-c2dacbcc90eb.png)

**Comment by manolobevia** [comment]
> Looking at this again, I think the bigger problem is that we have duplicate staff in the central list (and it's not limited to just the Makerspace). Maybe that's what you're referring to?

@cappadona @sew268 @eeh53 @kevinkidwell That is another issue. 
Ex: Prior the new staff vue interface, on the Rare site we have Eric Acree displayed as part of the staff of Rare unit. Custom loops plugin query all the staff post and display then. The plugin doesn't care what unit you have selected on the ACF staff group. It will displays all the staff. Which is right.

With the new staff interface, If we want to display Eric in rare unit, we need to select Rare as unit. Also, if we want to display Eric in Africana, we need to select Africana as unit. When you go to central, this display Eric twice. 

Eric should displays only once as part of Africana unit in the Africana unit and the Central ULS-Library, but Rare unit also wants to display Eric too.

**Comment by manolobevia** [comment]
Same thing with these librarians: Bonna Boettcher, Tamar Dougherty, Rachel Holder, Leah McEwen, Christina Sheley, and Henrik Spoon.

**Comment by cappadona** [comment]
Okay. We'll leave this issue for Makerspace. I created https://github.com/cul-it/wp-cul-theme-culu/issues/156 for managing staff in multiple units.

I stand by my [original opinion](https://github.com/cul-it/uls/issues/1034#issuecomment-841310518): It's not a high priority, since we're not rendering all CUL staff anywhere just yet.

Depending on the consensus here, the fix is easy (don't push/distribute staff from Makerspace site), but also kind of depends on our decision re: managing staff in multiple units (https://github.com/cul-it/wp-cul-theme-culu/issues/156) since we don't have any dedicated Makerspace staff.

**Comment by manolobevia** [comment]
I agree.

**Comment by JesiBuell79** [comment]
I don't think this is an issue anymore
